app/views.py

```diff
@@ -0,0 +1,79 @@
+from flask import Flask, jsonify, request
+from app.email import Emails
+from app.validator import Validation
+from flask_jwt_extended import (
+    JWTManager, jwt_required, create_access_token, get_jwt_identity)
+
+
+app = Flask(__name__)
+emails = Emails()
+validator = Validation()
+JWT = JWTManager(app)
+app.config['JWT_SECRET_KEY'] = 'super-secret'
+
+
+@app.route('/', methods=['GET'])
+def index():
+    """opening route."""
+    return jsonify({
+        "status": 200, 'message': 'welcome to Epic Email.'
+        }), 200
+
+
+@app.route('/api/v1/users', methods=['POST'])
```

Comment:
This is not the endpoint for `signup` look through the doc and change it

Reply:
made changes